lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -167,6 +176,69 @@ class CategoryRenderer {
     element.appendChild(passedElem);
     return element;
   }
+
+  _renderAuditGroup(audits, tag) {
+      const auditGroupElem = this._dom.createElement('details', 'lh-audit-group');
+      const auditGroupHeader = this._dom.createElement('div', 'lh-audit-group__header');
+      auditGroupHeader.textContent = tag.title;
+
+      const auditGroupDescription = this._dom.createElement('div', 'lh-audit-group__description');
```

Comment:
Yea, we can rename the CSS if you need to.

"distinct reusable component"....interesting idea :)